workflow-ui/compose/src/main/java/com/squareup/workflow1/ui/compose/RenderAsState.kt

```diff
@@ -162,6 +175,12 @@ private class WorkflowRuntimeState<PropsT, OutputT : Any, RenderingT>(
   private val renderingState = mutableStateOf<RenderingT?>(null)
   private val propsFlow = MutableStateFlow(initialProps)
 
+  /**
+   * The actual scope used to run the workflow. It has a child [Job] of the incoming scope so
+   * we can cancel the runtime without cancelling the incoming scope.
+   */
+  private val workflowScope = workflowScope + Job(parent = workflowScope.coroutineContext.job)
```

Comment:
The `job` extension property throws if the context has no job. In this case, we don't actually care if the parent doesn't have a job – in that case, this will be the root job, and that's fine.
```suggestion
  private val workflowScope = workflowScope + Job(parent = workflowScope.coroutineContext[Job])
```